build/compliance-build.yml

```diff
@@ -11,7 +11,7 @@ variables:
 jobs:
```

Comment:
>jobs [](start = 0, length = 4)

Now that the compliance checks are part of the signed build, I'm fairly confident that this yml file is no longer used. I'll try to confirm this theory